package proto_ff;

import "yd_fieldoptions.proto";


message E_CollectCollectAttributDesc
{
	optional int32 m_num = 1[(yd_fieldoptions.field_cname) = "数量"];
	optional int32 m_id = 2[(yd_fieldoptions.field_cname) = "ID"];
}

message E_CollectCollect
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "序号"];
	optional string m_name = 2[(yd_fieldoptions.field_cname) = "真甲名称", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 m_lv = 3[(yd_fieldoptions.field_cname) = "开启等级"];
	optional int32 m_group = 4[(yd_fieldoptions.field_cname) = "分组"];
	optional string m_stagetitle = 5[(yd_fieldoptions.field_cname) = "阶段标题", (yd_fieldoptions.field_bufsize) = 60];
	optional string m_position = 6[(yd_fieldoptions.field_cname) = "装备部位", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 m_grade = 7[(yd_fieldoptions.field_cname) = "要求阶数"];
	optional int32 m_quality = 8[(yd_fieldoptions.field_cname) = "要求品质"];
	optional int32 m_star = 9[(yd_fieldoptions.field_cname) = "要求星级"];
	optional int32 m_skillid = 10[(yd_fieldoptions.field_cname) = "激活技能"];
	repeated E_CollectCollectAttributDesc m_attribut = 11[(yd_fieldoptions.field_cname) = "套装属性", (yd_fieldoptions.field_arysize) = 4];
}


message Sheet_CollectCollect
{
	repeated E_CollectCollect E_CollectCollect_List = 1[(yd_fieldoptions.field_arysize)=60];
}

message E_CollectAttributetpyeAttributeDesc
{
	optional int32 m_type = 1[(yd_fieldoptions.field_cname) = "ID"];
	optional int32 m_value = 2[(yd_fieldoptions.field_cname) = "值"];
}

message E_CollectAttributetpye
{
	optional int64 m_id = 1[(yd_fieldoptions.field_cname) = "id"];
	repeated E_CollectAttributetpyeAttributeDesc m_attribute = 2[(yd_fieldoptions.field_cname) = "基础属性", (yd_fieldoptions.field_arysize) = 3];
}


message Sheet_CollectAttributetpye
{
	repeated E_CollectAttributetpye E_CollectAttributetpye_List = 1[(yd_fieldoptions.field_arysize)=200];
}

message E_CollectConst
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "id"];
}


message Sheet_CollectConst
{
	repeated E_CollectConst E_CollectConst_List = 1[(yd_fieldoptions.field_arysize)=20];
}
